uqcs/templates/form.mako

```diff
@@ -1,176 +1,243 @@
+        <input
+          type="text"
+          class="form-control"
+          id="fnameInput"
+          placeholder="First Name"
+          name="fname"
+          required="true"
+        />
       </div>
-      <div class="radio">
-        <label>
-          <input name="gender" type="radio" value="null" data-bind="checked: gender" checked>
-          Other / Prefer not to disclose
-        </label>
+      <div class="form-group">
+        <label for="lnameInput">Last Name<span class="reqstar">*</span></label>
+        <input
+          type="text"
+          class="form-control"
+          id="lnameInput"
+          placeholder="Last Name"
+          name="lname"
+          required="true"
+        />
       </div>
-    </div>
-  </div>
-  <div class="form-group">
-    <label for="memberType">Member Type <span class="reqstar">*</span></label>
-    <div class="checkbox">
-        <label>
-          <input name="student" type="checkbox" id="studentCheckbox">
-          Are you currently a student?
-        </label>
-    </div>
-  </div>
-  <div id="student-form-section" style="display:none;">
-    <div class="form-group">
-      <label for="student-no">Student Number <span class="reqstar">*</span></label>
-      <input type="number" name="student-no" class="form-control" id="studentNo" placeholder="43108765">
-      <p class="help-block">8 digits, no 's'</p>
-    </div>
-    <div class="form-group">
-      <label>Domestic or International</label>
-      <div class="radio">
-        <label>
-          <input type="radio" value="domestic" name="domORint">
-          Domestic
-        </label>
+      <div class="form-group">
+        <label for="emailInput"
+          >Email address <span class="reqstar">*</span></label
+        >
+        <input
+          name="email"
+          type="email"
+          class="form-control"
+          id="emailInput"
+          placeholder="Email"
+          required="true"
+        />
       </div>
-      <div class="radio">
-        <label>
-          <input type="radio" value="international" name="domORint">
-          International
-        </label>
+      <div class="form-group">
+        <label>Gender</label><br />
+        <div class="btn-group-toggle" data-toggle="buttons">
+          <div class="btn-group">
+            <label class="btn btn-primary"> 
+              <input name="gender" type="radio" value="M" /> Male 
+            </label>
+            <label class="btn btn-primary">
+              <input name="gender" type="radio" value="F" /> Female
+            </label>
+          </div>
+          <br />
+          <div class="btn-group" style="padding-top:10px">
+            <label class="btn btn-primary">
+              <input
+                name="gender"
+                type="radio"
+                value="null"
+                data-bind="checked: gender"
+                checked
+              />
+              Other / Prefer not to disclose
```

Comment:
thoughts on putting this on the same line as Male/Female? Might wreck the symmetry though.

Reply:
Yeah, this is why I left it off the same line. It also means it scales nicer to mobile devices.